### ISSUE TITLE
It freezes when i try to apply damage that brings the creature HP below or exactly equal to zero

### ISSUE THREAD

**Issue by Sara-nox** [javalent/initiative-tracker#97]
Whenever I apply damage that brings one creature or party member HP below or equal to zero it freezes the "Initiative tracker" panel.
When it happens can't interact with it anymore, I have to close and restart Obsidian to be able to use the plugin again.

---

I have Obsidian 1.1.9
Windows 11
Plugins installed: Advanced Tables, AidenLx's Folder Note+folderv component, Dataview, Dice Roller, Fantasy Calendar, Folder Note Core, Icon Folder, Initiative Tracker, Minimal Theme Settings, Obsidian Leaflet, Style settings, Timelines, TTRPG Statoblocks.

---

I loved Dice Roller, Leaflet, Fantasy Calendar and TTRPG statblocks I don't know how I would survive without them, thank youso much for your work! :)

**Comment by valentine195** [comment]
I can't recreate this unfortunately, can you open the developer console and check for errors? Ctrl/shift/i on windows.

**Comment by Dragander** [comment]
I have the same problem. As soon as I bring a creature to or below 0 hit points the following error message shows up in the console and everything stops working so that I have to restart Obsidian again:

![grafik](https://user-images.githubusercontent.com/46267364/221338550-b7ab40c6-cc71-48af-a439-9d05aa96d017.png)

The line it spits out where the error occurs is this one:

`let t0_value = ctx[0].name + "";`

Accompanying error message:
>  Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'name')

___
I use version v1.1.15 of Obsidian on a Windows 10 machine with the ITS theme and the following community plugins:
Advanced Tables, Annotator, Auto Note Mover, Banners, Checklist, cMenu, Custom Frames, DataView, Dice Roller, Excalidraw, Execute Code, Extract url content, Fantasy Calendar, Fantasy Statblocks, Folder Note, Homepage, Icon Finder, Icons, Image Toolkit, Kanban, Markdown Formatting Assistant, Media Extended, Obsidian Leaflet, Pandoc Plugin, Paste URL into selection, Quick Add, Reading Time, Style Settings, Templater, Timelines, Various Complements and of course Initiative Tracker

**Comment by landerson9** [comment]
I may have found the problem ? I was having the same issue, freezing up when trying to drop creatures to 0 HP, and I realized the culprit was the statuses in the initiative tracker settings. I play with my group in French so I had translated all my statuses, but it turns out that meant that the plug-in was getting stuck in a loop when I dropped creatures to 0 HP and the "apply unconscious status at zero HP" setting was turned on. The status "inconscient" existed, the French translation of "unconscious", but the plugin could not identify that it was the equivalent as there is no way to manipulate that setting, so it got stuck looking for the status with the specific spelling "unconscious"...which didn't exist in my translated statuses. Anyway, I made sure that there was a real "unconscious" status that followed the same spelling and lower/uppercase as indicated in the text for the "apply unconscious status at zero HP" tick box in the settings of the plugin and everything runs smoothly now !
